es/6/index.json

```diff
@@ -0,0 +1,271 @@
+                "¡¿Un Asesino de CryptoKitties?!",
+                "Resulta que los CryptoZombies {love} alimentarse de CryptoKitties.",
+                "Cuando un CryptoZombie se alimenta de un CryptoKitty, sus ADN se fusionan y el CryptoKitty se transforma en un CryptoZombie.",
+                "¡Sigue adelante, selecciona un CryptoKitty para atacar, y comprueba que GatoZombi consigues!",
+                "Conceptos de Solidity Dominados",
+                "Para completar la Lección 2 de la Escuela de Programación de CryptoZombies, tu amigo ha construido una {zombieCardBattle} y ha aprendido sobre:"
+            ],
+            "achievements": [
+                "Los elementos básicos sobre creación de un juego multijugador",
+                "Interactuar con otros contratos de la red de Ethereum",
+                "Organizar proyectos de Solidity más extensos en multiples contratos"
+            ],
+            "callToAction": [
+                "¿Quiéres crear tu propio CryptoZombie y añadirlo a tus filas?",
+                "¡Aprende como crear tus propios juegos en Ethereum GRATIS con CryptoZombies! Empieza ahora:"
+            ]
+        },
+        "lesson3": {
+            "achievementPreamble": [
+                "¡Tu amigo ha completado la Lección 3 de CryptoZombies, y ha mejorado su ejército zombi!",
+                "Para completar la Lección 3 de CryptoZombies, tu amigo ha aprendido sobre:"
+            ],
+            "achievements": [
+                "Como crear contratos inteligentes actualizables",
+                "Asegurar contratos con la titularidad de ellos",
+                "Gas y optimización del gas en Ethereum",
+                "Modificadores de funciones y comprobación de seguridad"
+            ],
+            "callToAction": [
+                "¿Quiéres crear tu propio CryptoZombie y añadirlo a tus filas?",
+                "¡Aprende como crear tus propios juegos en Ethereum GRATIS con CryptoZombies! Empieza ahora:"
+            ]
+        },
+        "lesson4": {
+            "achievementPreamble": [
+                "¡Tu amigo ha completado la Lección 4 de CryptoZombies, y ha derrotado al malvado zombi IOTA!",
+                "Sigue adelante y pruébalo tu mismo a la izquierda. ¡Usa el ejército de tu amigo para entrar en la batalla!",
+                "Para completar la Lección 4 de CryptoZombies, tu amigo ha aprendido sobre:"
+            ],
+            "achievements": [
+                "Funciones de pago, y como ganar dinero con tus juegos basados en Ethereum",
+                "Retirar ETH de los contratos inteligentes",
+                "Generar numeros aleatorios y la seguridad en Ethereum",
+                "¡Y más!"
+            ],
+            "callToAction": [
+                "¿Quiéres crear tu propio CryptoZombie y añadirlo a tus filas?",
+                "¡Aprende como crear tus propios juegos en Ethereum GRATIS con CryptoZombies! Empieza ahora:"
+            ]
+        }
```

Comment:
Thanks for the previous fix, here we also need JSON data for lesson 5 and 6 :)

Reply:
Excuse me jejejej. Now it's done.